pkg/eval/vals/equal.go

```diff
@@ -4,6 +4,8 @@ import (
 	"reflect"
 )
 
+const float64EqualityThreshold = 1e-9
```

Comment:
Remove

Reply:
Even if I implement `Approximate()`, as you suggested in a different comment, we still need this constant or something like it. So I don't understand this comment.

Reply:
You already the same constant in testutils.go, with a different value :)

Reply:
Gah! Sorry, I thought I had removed that already.